app/modules/users/components/inventory_or_listing_nav.svelte

```diff
@@ -1,47 +1,50 @@
 <script>
   import { i18n } from '#user/lib/i18n'
-  import app from '#app/app'
   import { isOpenedOutside } from '#lib/utils'
-  import { isNumber } from 'underscore'
+  // import { isNumber } from 'underscore'
 
-  export let userModel, groupModel, currentSection
+  export let user, group, currentSection
 
-  let inventoryPathname, listingsPathname, listingsCount
+  let inventoryPathname, listingsPathname, listingsCount, navEl
 
-  const aggregateMembersListsCount = () => {
-    const groupUsersModels = groupModel.members.models
-
-    const sumsListingsCount = (sum, user) => {
-      const newCount = user.get('listingsCount')
-      if (isNumber(newCount)) sum += newCount
-      return sum
-    }
-    return groupUsersModels.reduce(sumsListingsCount, 0)
+  if (user) {
+    ;({ inventoryPathname, listingsPathname } = user)
+  } else if (group) {
+    ;({ inventoryPathname, listingsPathname } = group)
   }
 
-  if (userModel) {
-    ;({ inventoryPathname, listingsPathname } = userModel.toJSON())
-  } else if (groupModel) {
-    ;({ inventoryPathname, listingsPathname } = groupModel.toJSON())
-  }
+  // user?.waitingForInventoryStats?.then(() => {
+  //   listingsCount = user.get('listingsCount')
+  // })
+  // group?.waitForMembers?.then(() => {
+  //   listingsCount = aggregateMembersListsCount()
+  // })
+
+  // const aggregateMembersListsCount = () => {
+  //   const groupUsersModels = group.members.models
 
-  userModel?.waitingForInventoryStats?.then(() => {
-    listingsCount = userModel.get('listingsCount')
-  })
-  groupModel?.waitForMembers?.then(() => {
-    listingsCount = aggregateMembersListsCount()
-  })
+  //   const sumsListingsCount = (sum, user) => {
+  //     const newCount = user.get('listingsCount')
+  //     if (isNumber(newCount)) sum += newCount
+  //     return sum
+  //   }
+  //   return groupUsersModels.reduce(sumsListingsCount, 0)
+  // }
 
   const showSection = (e, section) => {
     if (isOpenedOutside(e)) return
     currentSection = section
-    app.vent.trigger('show:inventory:or:listing:section', { section, userModel, groupModel })
+    if (currentSection === 'inventory') {
+      app.navigate(inventoryPathname, { pageSectionElement: navEl })
+    } else {
+      app.navigate(listingsPathname, { pageSectionElement: navEl })
```

Comment:
From a listings browser layout, when i click on a listing (aka. i get extracted from user layout context) then, if i click on the browser precedent button, i expect to be back to the listings page, but instead the inventory browser (items) is displayed with a URL `/users/adamsberg/lists`

Reply:
fixed by 4e4562a92